src/middlewared/middlewared/plugins/disk_/wipe.py

```diff
@@ -54,7 +54,7 @@ def _wipe(self, data):
     @accepts(
         Str('dev'),
         Str('mode', enum=['QUICK', 'FULL', 'FULL_RANDOM'], required=True),
-        Bool('synccache', default=True),
```

Comment:
We shouldn't change the name of the variable as existing users consuming this via API would be affected by this change

Reply:
These are positional arguments, they are never referenced by their names